src/ui/components/DevTools.tsx

```diff
@@ -106,7 +106,7 @@ function Body() {
   }, [sidePanelCollapsed]);
 
   return (
-    <div className="vertical-panels pr-1">
```

Comment:
There have always been equal-sized gaps between columns 1 and 2, and then again between columns 2 and 3, and again between columns 3 and the right edge of the application chrome. In the past we used tailwind class w-2, and #9886 moved everything to w-1.

Old version: w-2, w-2, w-2
After #9886: w-1, w-1, w-1
This change: w-1, w-1, w-0

I think we should put this back. There are usability reasons why you don't push elements against the edge of your application canvas, but the UI is now more inconsistent. Additionally, Jason's misgivings in #9886 were because he thought this new layout would be too cramped, so going from w-2 (8px) to w-0 is making it even more cramped.

Reply:
I don't think adding arbitrary padding to one side of one of our panel sis the right approach to the scenario you're describing.